envoy/ssl/context_config.h

```diff
@@ -118,7 +142,7 @@ class ClientContextConfig : public virtual ContextConfig {
   virtual const std::string& signingAlgorithmsForTest() const PURE;
 };
 
-using ClientContextConfigPtr = std::unique_ptr<ClientContextConfig>;
+using ClientContextConfigPtr = std::shared_ptr<ClientContextConfig>;
```

Comment:
Why is this changing from a unique_ptr to shared_ptr?

Reply:
fixed